### PR TITLE
Update CLI Documentation with GraphQL Schema Generation Command

### PR DESCRIPTION
**Pull request by DimuthuMadushan**
Fixes: [#3737](https://github.com/ballerina-platform/ballerina-standard-library/issues/3737)

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: swan-lake/learn-the-platform/ballerina-tooling/cli-documentation/cli-commands.md [hunk 1]
```diff
@@ -74,8 +74,8 @@ COMMANDS
         format          Format Ballerina source files
         grpc            Generate the Ballerina sources for a given Protocol
                         Buffer definition
-        graphql         Generate Ballerina client sources
-                        for a given GraphQL schema (SDL) and GraphQL queries
+        graphql         Generate Ballerina client sources for a GraphQL config file
```
Comment (praneesha) [inline swan-lake/learn-the-platform/ballerina-tooling/cli-documentation/cli-commands.md:77]:
```suggestion
        graphql         Generate the Ballerina client sources for a given GraphQL config file
```
Changed to preserve consistency with line numbers 79-81

File: swan-lake/learn-the-platform/ballerina-tooling/cli-documentation/cli-commands.md [hunk 2]
```diff
@@ -74,8 +74,8 @@ COMMANDS
...
+                        and generate GraphQL schema for a Ballerina GraphQL service.
```
Comment (praneesha) [inline swan-lake/learn-the-platform/ballerina-tooling/cli-documentation/cli-commands.md:78]:
```suggestion
                        and generate the GraphQL schema for a given Ballerina GraphQL service.
```

File: swan-lake/learn-the-platform/ballerina-tooling/cli-documentation/graphql.md [hunk 3]
```diff
@@ -15,15 +15,33 @@ redirect_from:
...
-| `-i, --input`  | The `graphql-configuration-file-path` parameter specifies the path of the GraphQL config file (e.g., `graphql.config.yaml`) configured with GraphQL schemas (SDL) and GraphQL documents. This parameter is mandatory. You can point to the GraphQL config file by using the mandatory flag (`-i,   --input`).                                                                 |
-| `-o, --output` | The `output-location` parameter specifies the path of the output location of the generated files. This parameter is optional. If this parameter is not specified, the Ballerina files are generated at the same location from which the GraphQL command is executed. Optionally, you can point to another directory location by using the optional flag (`-o, --output`). |
-                                                                                                         
+| `-i, --input`  | The `input` parameter specifies the path of the GraphQL config file (e.g., graphql.config.yaml) configured with GraphQL schemas specified by Schema Definition Language and GraphQL documents. This parameter is mandatory.                                                                 |
```
Comment (praneesha) [inline swan-lake/learn-the-platform/ballerina-tooling/cli-documentation/graphql.md:26]:
```suggestion
| `-i, --input`  | The `input` parameter specifies the path of the GraphQL config file (e.g., `graphql.config.yaml`) configured with GraphQL schemas specified by the Schema Definition Language and GraphQL documents. This parameter is mandatory.                                                                 |
```

File: swan-lake/learn-the-platform/ballerina-tooling/cli-documentation/graphql.md [hunk 4]
```diff
@@ -15,15 +15,33 @@ redirect_from:
...
+| `-o, --output` | The `output` parameter specifies the path of the output location of the generated files. This parameter is optional. If this parameter is not specified, the Ballerina files will be generated at the same location from which the GraphQL command is executed. |
```
Comment (praneesha) [inline swan-lake/learn-the-platform/ballerina-tooling/cli-documentation/graphql.md:27]:
```suggestion
| `-o, --output` | The `output` parameter specifies the path of the output location of the generated files. This parameter is optional. If this parameter is not specified, the Ballerina files will be generated at the same location in which the GraphQL command is executed. |
```

File: swan-lake/learn-the-platform/ballerina-tooling/cli-documentation/graphql.md [hunk 5]
```diff
@@ -15,15 +15,33 @@ redirect_from:
...
+bal graphql [-i | --input] <ballerina-graphql-service-file-path>
+            [-o | --output] <output-location>
+            [-s | --service] <service-base-path>
+```
+
+The command-line arguments below can be used with the command for each particular purpose as described below.
+
+| Argument        | Description                                                                                                                                                                                                                                                     |
+|-----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `-i, --input`   | The `input` parameter specifies the path of the Ballerina GraphQL service file (e.g., service.bal). This parameter is mandatory.                                                                                                                                |
```
Comment (praneesha) [inline swan-lake/learn-the-platform/ballerina-tooling/cli-documentation/graphql.md:43]:
```suggestion
| `-i, --input`   | The `input` parameter specifies the path of the Ballerina GraphQL service file (e.g., `service.bal`). This parameter is mandatory.                                                                                                                                |
```

File: swan-lake/learn-the-platform/ballerina-tooling/cli-documentation/graphql.md [hunk 6]
```diff
@@ -15,15 +15,33 @@ redirect_from:
...
+| `-o, --output`  | The `output` parameter specifies the output location of the generated GraphQL schema files. This parameter is optional. If this parameter is not specified, the schema files will be generated at the same location from which the GraphQL command is executed. |
```
Comment (praneesha) [inline swan-lake/learn-the-platform/ballerina-tooling/cli-documentation/graphql.md:44]:
```suggestion
| `-o, --output`  | The `output` parameter specifies the output location of the generated GraphQL schema files. This parameter is optional. If this parameter is not specified, the schema files will be generated at the same location in which the GraphQL command is executed. |
```

File: swan-lake/learn-the-platform/ballerina-tooling/cli-documentation/graphql.md [hunk 7]
```diff
@@ -15,15 +15,33 @@ redirect_from:
...
+| `-o, --service` | The `service` parameter specifies the base path of the Ballerina GraphQL service which the schema is needed to be generated. This parameter is optional. If this parameter is not specified, schemas will be generated for each of the GraphQL services in the given file. |
```
Comment (praneesha) [inline swan-lake/learn-the-platform/ballerina-tooling/cli-documentation/graphql.md:45]:
```suggestion
| `-o, --service` | The `service` parameter specifies the base path of the Ballerina GraphQL service of which the schema needs to be generated. This parameter is optional. If this parameter is not specified, schemas will be generated for each of the GraphQL services in the given file. |
```

**Review by ThisaruGuruge — APPROVED** [review]
LGTM

**Review by praneesha — APPROVED** [review]

**Comment by ThisaruGuruge** [comment]
@praneesha Shouldn't we merge this to the master branch as well?

**Pull request merged**